Formula/jupyterlab.rb

```diff
@@ -28,8 +28,10 @@ class Jupyterlab < Formula
   end
 
   resource "appnope" do
-    url "https://files.pythonhosted.org/packages/e9/bc/2d2c567fe5ac1924f35df879dbf529dd7e7cabd94745dc9d89024a934e76/appnope-0.1.2.tar.gz"
-    sha256 "dd83cd4b5b460958838f6eb3000c660b1f9caf2a5b1de4264e941512f603258a"
+    on_macos do
```

Comment:
I think this might break `brew update-python-resources` (which is also called with `bump-formula-pr`). If so, we should move this logic to the `install` method.

@Rylan12 can correct me here if I'm mistaken.

Reply:
Yeah, it will break `update-python-resources`. Not totally sure if it will do it silently or just cause the command to fail to run, but I'm pretty sure it will be the latter. It definitely won't keep the `on_macos` and do what you want, though.

Reply:
Thanks for clarifying this, I've just removed `appnope` from the dependencies array in the `install` block in Linux.